src/main/webapp/encounters/encounter.jsp

```diff
@@ -1293,9 +1294,17 @@ if(CommonConfiguration.showProperty("showCountry",context)){
            	%>
 
        marker = new google.maps.Marker({
-    	   icon: 'https://chart.googleapis.com/chart?chst=d_map_pin_letter&chld=<%=markerText%>|<%=haploColor%>',
+    	   //icon: 'https://chart.googleapis.com/chart?chst=d_map_pin_letter&chld=<%=markerText%>|<%=haploColor%>',
+    	   icon: " ",
```

Comment:
am guessing the space must be needed here (versus just dropping the `icon` element from the json entirely).

Reply:
I believe the space is needed according to some reading I did on this issue. But I also didn't test without it.